Dockerfile

```diff
@@ -11,6 +11,15 @@ RUN apt-get update && apt-get install -y build-essential git
 COPY requirements/core.txt requirements/core.txt
 RUN pip3 install -r requirements/core.txt
 
+# This is below the preceding layer to prevent Docker from rebuilding the
+# previous layer (forcing a reload of dependencies) whenever the
+# status of a local service library changes
+ARG service_lib_dir=NO_SUCH_DIR
+
+# Install a local harmony-service-lib-py if we have one
+COPY deps ./deps/
+RUN if [ -d deps/${service_lib_dir} ]; then echo "OK"; pip install -e deps/${service_lib_dir}; fi
```

Comment:
Nit: I know this was what the existing code did, but if we're going to print something it should be informative rather than "OK". Maybe something like "Installing from local copy of harmony-service-lib"?